Src/Processor/Standard/StandardScore.cpp

```diff
@@ -127,8 +127,10 @@ void StandardScore::computeAimValue(const Beatmap& beatmap)
 		_aimValue *= 1.03f;
 
 	if ((_mods & EMods::Flashlight) > 0)
-		// Apply length bonus again if flashlight is on simply because it becomes a lot harder on longer maps.
-		_aimValue *= 1.45f * LengthBonus;
+		// Apply object-based bonus for flashlight.
+		_aimValue *= 1.0f + 0.35f * std::min(1.0f, static_cast<f32>(numTotalHits) / 250.0f) +
++        		(numTotalHits > 250 ? 0.3f * std::min(1.0f, static_cast<f32>(numTotalHits - 250) / 250.0f) +
++        		(numTotalHits > 500 ? static_cast<f32>(numTotalHits - 500) / 1200.0f : 0.0f) : 0.0f);
```

Comment:
what's up with the leading plus signs on these lines, looks like you pasted from a diff or something